app/views/govuk_publishing_components/components/docs/emergency_banner.yml

```
@@ -0,0 +1,47 @@
+name: Emergency banner
+description: A site-wide banner used in the event of an emergency
+body: |
+  There are three types of events that would lead GOV.UK to add an emergency banner to the top of each page on the web site; a notable death, a national emergency or a local emergency.
+
+  See the [opsmanual](https://docs.publishing.service.gov.uk/manual/emergency-publishing.html#adding-emergency-publishing-banners) for information about what the Emergency Banner is and when it should be deployed.
+
+accessibility_criteria: |
+  The links must:
+
+  * be keyboard focusable
+
```

Sorry, I've just spotted this - I think this is the last thing!

There's a `shared_accessibility_criteria` that can be used when a component uses links and/or buttons:

```suggestion
shared_accessibility_criteria:
- link
```

This can then show the full `link` accessibility criteria from:

https://github.com/alphagov/govuk_publishing_components/blob/22d85c2c96e98ac6534260c84f02f07d7cf38269/app/models/govuk_publishing_components/shared_accessibility_criteria.rb#L3-L18

---

oh that's useful!